pkgs/by-name/tr/trilium-next-desktop/package.nix

```diff
@@ -0,0 +1,130 @@
+{
+  stdenv,
+  lib,
+  unzip,
+  fetchurl,
+  fetchzip,
+  makeBinaryWrapper,
+  # use specific electron since it has to load a compiled module
+  electron_31,
+  autoPatchelfHook,
+  makeDesktopItem,
+  copyDesktopItems,
+  wrapGAppsHook3,
+  asar,
+}:
+
+let
+  pname = "trilium-next-desktop";
+  version = "0.90.12";
+
+  linuxSource.url = "https://github.com/TriliumNext/Notes/releases/download/v${version}/TriliumNextNotes-v${version}-linux-x64.zip";
+  linuxSource.sha256 = "0ji28l60wyzhjbi6g5845dnm763bvg7535zfgzcmfgwjs6zr6nfq";
+
+  darwinSource.url = "https://github.com/TriliumNext/Notes/releases/download/v${version}/TriliumNextNotes-v${version}-macos-x64.zip";
+  darwinSource.sha256 = "0jv80k7dk6gpyfj36iin6y7fk7qan4bya72f14jcgfla95wvk6ls";
+
+  meta = {
+    description = "Hierarchical note taking application with focus on building large personal knowledge bases";
+    homepage = "https://github.com/TriliumNext/Notes";
+    license = lib.licenses.agpl3Plus;
+    sourceProvenance = with lib.sourceTypes; [ binaryNativeCode ];
+    maintainers = with lib.maintainers; [
+      eliandoran
+      fliegendewurst
+    ];
+    mainProgram = "trilium";
+    platforms = [
+      "x86_64-linux"
+      "x86_64-darwin"
+    ];
+  };
+
+  linux = stdenv.mkDerivation rec {
+    inherit pname version meta;
+
+    src = fetchurl linuxSource;
+
+    nativeBuildInputs = [
+      unzip
+      makeBinaryWrapper
+      wrapGAppsHook3
+      copyDesktopItems
+      autoPatchelfHook
+      asar
+    ];
+
+    buildInputs = [
+      stdenv.cc.cc.lib
+    ];
+
+    desktopItems = [
+      (makeDesktopItem {
+        name = "Trilium";
+        exec = "trilium";
+        icon = "trilium";
+        comment = meta.description;
+        desktopName = "TriliumNext Notes";
+        categories = [ "Office" ];
+        startupWMClass = "trilium notes next";
```

Comment:
Can you double check please that the `StartupWMClass` is correct?

I believe on GNOME the WM class is case sensitive: https://www.reddit.com/r/emacs/comments/1dm0v6g/comment/l9sy73c/?utm_source=share&utm_medium=web3x&utm_name=web3xcss&utm_term=1&utm_content=share_button

For Debian installs, I've opted for `Trilium Notes Next` as the WM class, based on what I saw through the Gnome Looking Glass.